solid-acp.ttl

```diff
@@ -0,0 +1,273 @@
+prefix acp: <http://www.w3.org/ns/solid/acp#>
+prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+prefix owl: <http://www.w3.org/2002/07/owl#>
+prefix xsd: <http://www.w3.org/2001/XMLSchema#>
+prefix dc: <http://purl.org/dc/terms/>
+prefix vann: <http://purl.org/vocab/vann/>
+
+<http://www.w3.org/ns/solid/acp#>
+    a owl:Ontology ;
+    rdfs:label "Access Control Policy Language (ACP)"@en ;
+    rdfs:comment "The Access Control Policy Language (ACP) is a language for describing, controlling and granting access to resources."@en ;
+    rdfs:isDefinedBy acp: ;
+    rdfs:seeAlso <https://solid.github.io/authorization-panel/acp-specification/> ;
+    dc:issued "2022-04-28"^^xsd:date ;
+    vann:preferredNamespacePrefix "vann" ;
+    vann:preferredNamespaceUri "http://www.w3.org/ns/solid/acp#"^^xsd:anyURI .
+
+
+#################
+#   Classes
+#################
+acp:Context
+    a rdfs:Class ;
+    rdfs:label "Context"@en ;
+    rdfs:comment "Instances of the Context class describe instances of resource access."@en ;
+    rdfs:isDefinedBy acp: .
+
+acp:AccessControlResource
+    a rdfs:Class ;
+    rdfs:label "Access Control Resource"@en ;
+    rdfs:comment "Instances of the Access Control Resource (ACR) class connect resources to their Access Controls."@en ;
+    rdfs:isDefinedBy acp: .
+
+acp:AccessControl
+    a rdfs:Class ;
+    rdfs:label "Access Control"@en ;
+    rdfs:comment "Instances of the Access Control class connect Access Control Resources to their Policies."@en ;
+    rdfs:isDefinedBy acp: .
+
+acp:Policy
+    a rdfs:Class ;
+    rdfs:label "Access Policy"@en ;
+    rdfs:comment "Instances of the Policy class connect Access Controls to allowed and denied Access Modes as well as sets of Matchers describing instances of resource access."@en ;
+    rdfs:comment "In ACP, a policy MUST be considered satisfied if and only if: It references at least one matcher via a condition; and, At least one matcher it references matches the given context; and, All the conditions it defines are satisfied. Given that the acp:noneOf condition excludes matches, a policy without a satisfied allOf or anyOf condition is never satisfied."@en ;
+    rdfs:comment "An access mode MUST be granted over a resource if and only if in the set of policies mandating access over it: A satisfied policy allows it; and No satisfied policy denies it. ACP doesn't define its own access modes, instead it reuses access modes defined elsewhere."@en ;
```

Comment:
These three `rdfs:comment` values appear to be paragraphs, meant to be read in the order seen here, which is not guaranteed to be preserved.  Probably better to make these one line of Turtle with embedded newlines (and I suggest embedding 2 newlines between each 2 paragraphs of text, to maximize clarity when rendered for humans). 

I find the streaming `MUST ... if and only if ...` phrasing is not as clear as my revision to `MUST NOT ... UNLESS ...`.  The former phrasing could also be clarified by breaking up the stream with additional newlines and plain-text bulleting (usually best based on an 80-column screen assuming monospaced font display).

I've done both of these in my suggestion, along with some other edits to the text. If the newlines and/or MUST segment edits are rejected, please take care to review the other text edits.
```suggestion
    rdfs:comment "Instances of the Policy class connect Access Controls to allowed and denied Access Modes, as well as to sets of Matchers describing instances of resource access.\n\nIn ACP, a policy MUST NOT be considered satisfied UNLESS (1) it references at least one matcher via a condition; (2) at least one matcher it references matches the given context; and (3) all the conditions it defines are satisfied. Given that the acp:noneOf condition excludes matches, a policy without a satisfied acp:allOf or acp:anyOf condition is never satisfied.\n\nAn access mode MUST NOT be granted over a resource UNLESS in the set of policies mandating access over it (1) a satisfied policy allows it; and (2) no satisfied policy denies it. ACP doesn't define its own access modes; instead, ACP reuses access modes defined elsewhere."@en ;
```